PeerioTechnologies/peerio-icebear
# fix: migitate Unicode text direction confusion with file extensions

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/3350/fix-displaying-of-extensions-in-filenames-with-rtl
 
#### Testing instructions  

* Check that file names are displayed as before.
* See comment for the specific check.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
